app/main.py

```diff
@@ -1,4 +1,10 @@
 def get_human_age(cat_age: int, dog_age: int) -> list:
+    if not isinstance(cat_age, int) or not isinstance(dog_age, int):
```

Comment:
don't change main code and it is useless

Reply:
How should I implement the function so that if it receives the wrong data type, it throws the correct exception?